AnimalFactoryView.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using Fsd.Jacek.Cs2.Animals;
+
+namespace Fsd.Jacek.Cs2.MVC
+{
+    class AnimalFactoryView : BaseView
+    {
+        private AnimalFactoryController _controller = null;

```

Comment:
to nie do zaakceptowania - wszystko wie o wszystkim. Widok nie powinien wiedzieć o controlerze